spring-ai-core/src/main/java/org/springframework/ai/model/function/FunctionCallingOptions.java

```diff
@@ -44,10 +44,10 @@ static FunctionCallingOptions.Builder builder() {
 
 	/**
 	 * Function Callbacks to be registered with the ChatModel. For Prompt Options the
```

Comment:
Should this class be removed as well?

Reply:
Along with FuntionCallingOptions, there are still other FunctionCallback references which are being used by some of the models. These models are planned to be moved into community supported modules. Once this plan is complete (hopefully, soon), we will have a follow up PR to clean up removal of rest of the deprecated classes and references.